evalme/metrics.py

```diff
@@ -163,7 +163,11 @@ def symmetrize(a, b):
 
         def clipped(s):
             if s > 1 or s < 0:
-                logger.error(f'Error in project {project}. Matching score {s} is not within [0, 1] interval '
+                if isinstance(project, dict):
+                    project_id = project.get('id')
+                else:
```

Comment:
maybe also check for `hasattr(project, 'id')` ?

Reply:
Removed obsolete checks for object, leaving only dict typed project